### ISSUE TITLE
VSCode `${relativeFile}` in launch.json does not detect filename rename casing change

### ISSUE THREAD

**Issue by ghiscoding** [microsoft/vscode#180754]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.78.0 insiders
- OS Version: Windows 10 - 21H2

Steps to Reproduce:

1. copy a file with a file that is named as camelCase, then rename the file to be PascalCase with the same name in Windows 10, then start any form of Launch action (in my use case, I start a Vitest debugger) and you will find that the `${relativeFile}` is still pointing to the original name as camelCase instead of the new rename of PascalCase. It seems that VSCode does not detect the filename change when only the casing changes, if I rename the file back to camelCase then my debugger start working again. It's also not related to git since I have nothing staged or committed yet 

![image](https://user-images.githubusercontent.com/643976/234121642-9a3db14d-ed19-45a1-9eb8-ab90977938ee.png)

my `launch.json` config is the following

```json
{
      "type": "node",
      "request": "launch",
      "name": "Vitest Current Spec File",
      "autoAttachChildProcesses": true,
      "skipFiles": ["<node_internals>/**", "**/node_modules/**"],
      "program": "${workspaceRoot}/node_modules/vitest/vitest.mjs",
      "args": ["run", "${relativeFile}"],
      "smartStep": true,
      "console": "internalConsole"
}
```

2.  the only way to have my debugger working at the moment is to rename the file as camelCase and not forget to rename it as PascalCase when doing my commit and PR

**Comment by alexr00** [comment]
@ghiscoding if you reload the VS Code window does it pick up the change?

**Comment by ghiscoding** [comment]
nope, it doesn't seem to ever catch the change. I also tried to play with Git and that has no effect, but even then it's not an issue related to Git. It should be easy enough for anyone with Windows to try it

**Comment by alexr00** [comment]
I'm having a hard time debugging this issue because of https://github.com/microsoft/vscode/issues/180764. I'll take another look after https://github.com/microsoft/vscode/issues/180764 is fixed.
